src/components/utils/useLayer.ts

```diff
@@ -28,8 +35,13 @@ export function useLayer({
         onOutsideClick,
         onClose,
         contentRefs,
+        idle: frozenIdle,
+        idlePriority: frozenIdlePriority,
+        idleTimeout: frozenIdleTimeout,
```

Comment:
Sorry, looked at it again and I think it was ok at first. Let's restore it as it was.
```suggestion
        idle,
        idlePriority,
        idleTimeout,
```